zinit-autoload.zsh

```diff
@@ -3246,7 +3246,8 @@ EOF
         setopt localoptions extendedglob nokshglob noksharrays
         builtin cd -q "${ZINIT[SNIPPETS_DIR]}"
         local -a list
-        list=( "${(f@)"$(LANG=en_US.utf-8 tree -L 3 --charset utf-8)"}" )
+        local -x LANG=en_US.utf-8
```

Comment:
Is this required by `exa`?

Reply:
Actually no. I took the issue's suggestion at face value, but as far as I can tell from exa's source code, it doesn't rely on locale for sorting.